Create ui-c8y-1024-10-3-fixed-the-inline-editing-radio-button-selection-graft-release-cd.md (#4980)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1024-10-3-fixed-the-inline-editing-radio-button-selection-graft-release-cd.md

---
date: ""
title: Fixed radio button not properly selected in configuration inline editing
product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-6805
version: 1024.10.3
---

> When editing device properties inline in the device management app, radio button selections were not being saved correctly. This issue has been fixed, and radio button selections now work as expected when you edit them inline. Users can now reliably update radio button fields directly in the device management interface without encountering selection errors.

Reviewer: ```suggestion
In the Device Management app, when using inline editing in the create/edit view of a configuration, the radio button was not properly selected. This has been fixed.
```

---
date: ""

> title: Fixed the inline editing radio button selection. [GRAFT][release/cd]

Reviewer: ```suggestion
title: Fixed radio button not properly selected in configuration inline editing
```
